spec.html

```diff
@@ -7613,13 +7613,12 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     <p>Jobs are scheduled for execution by ECMAScript host environments. This specification describes the host hook HostEnqueuePromiseJob to schedule one kind of job; host environments may define additional abstract operations which schedule jobs. Such operations accept a Job Abstract Closure as the parameter and schedule it to be performed at some future time. Their implementations must conform to the following requirements:</p>
 
     <ul>
-      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must:
+      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must perform the following:
         <ol>
-          <li>Push an execution context onto the execution context stack.</li>
           <li>Perform any implementation-defined preparation steps.</li>
           <li>Call the Abstract Closure.</li>
           <li>Perform any implementation-defined cleanup steps.</li>
-          <li>Pop the previously-pushed execution context from the execution context stack.</li>
+          <li>Assert: The execution context stack is empty.</li>
```

Comment:
Nit: Assertions are meant to be strictly editorial. Since the intent here is to put an invariant on hosts, we should phrase it differently. Perhaps after "the implementation must perform the following" we could add "after which the execution context stack must be empty".

Reply:
Will change.